doc/DeveloperGuide.md

```diff
@@ -88,6 +88,33 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename Tag
+
+**MSS**
+
+1. User requests to list all existing tags
+2. AddressBook shows a list of existing tags
+3. User requests to rename a tag in the list
+4. AddressBook renames the tag
+5. Addressbook updates the name change onto the corresponding tag of each person in the Addressbook
```

Comment:
possibly put both 4 & 5 together or refine the work flow to include user action between 4 and 5
use cases capture the back and forth interaction between the system and the user.